### ISSUE TITLE
Windows 10 Black Screen and Missing File Explorer

### ISSUE THREAD

**Issue by RestMx** [Sycnex/Windows10Debloater#423]
I opened powershell, copied the quick link, and received the GUI without a hitch. Afterward, I opened the blocklist and remove a few minor programs I didn't want deleted. For example, I kept MS.Paint. I clicked the bloat feature with the blocklist option, and the went through the various options of disabling cortana, one drive, and removing the registry keys.

After this happened, the background turned black. The file explorer no longer function. The applications I had open remained as small tabs which could be expanded. I tried restoring the registry keys to see if that fix something. The command ran, but no changes occurred. I cannot reach the settings to perform a factory reset. I am unsure where to go for here. Input would be appreciated.

**Comment by tort-oise** [comment]
Are you still having issues with this?

**Comment by RestMx** [comment]
No. I took a step back, had a coffee, and then followed troubleshooting guidelines to determine the issue. I resolved the problem by re-enabling my graphics card and then restarting the computer. I've not been able to determine what went wrong. The simplest explanation is probably I click something during the whitelisting process. Soon after, I got distracted with working pseudo code and forgot to close this ticket. Thank you for reaching out all the same.
